.github/workflows/terraform-vanilla-test.yaml

```diff
@@ -0,0 +1,55 @@
+name: Terraform Vanilla Test
+on:
+  # Run on PR configuration
+  push:
+    paths:
+      - deployments/vanilla/terraform/**
+      - iaac/terraform/kubeflow-components/**
+      - iaac/terraform/kubernetes-addons/**
+      - iaac/terraform/utils/**
+    branches:
+      - main
+  schedule:
```

Comment:
I was thinking to run the one with schedule in a different region, so lets remove schedule from this job for now and find a way to check its running from schedule and set a different region or create a separate job

Reply:
done

Reply:
Awesome!!